.github/workflows/maven-publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           distribution: temurin # Temurin is a distribution of adoptium
           java-version: 21
       - uses: actions/checkout@v4
-      - uses: aws-actions/configure-aws-credentials@v4.0.1
+      - uses: aws-actions/configure-aws-credentials@v4.0.2
```

Comment:
Suggest just use `v4` instead of specific version unless you have specific requirements here.

Thanks!

Reply:
If you use v4 it will automatically link to the latest v4.x.x.